Cython/Compiler/Code.py

```diff
@@ -1423,23 +1423,33 @@ def new_string_const_cname(self, bytes_value):
         value = bytes_value.decode('ASCII', 'ignore')
         return self.new_const_cname(value=value)
 
-    def new_num_const_cname(self, value, py_type):
+    def make_unique_const_cname(self, format_str):  # type: (str) -> str
+        used = self.const_cnames_used
+        cname = format_str % ""
+        counter = 0
+        while used.get(cname):
+            counter += 1
+            cname = format_str % counter
```

Comment:
`used` is a dict that maps values to counts. Please maintain that property.
```suggestion
        while cname in used:
            counter = used[value] = used[value] + 1
            cname = format_str % counter
```